app/components/insights-privacy-selector.js

```diff
@@ -10,6 +10,8 @@ export default Component.extend({
   tagName: 'span',
   classNames: ['insights-privacy-selector'],
   classNameBindings: ['isPrivateViewable:insights-privacy-selector--selectable'],
+  attributeBindings: ['testSelector:data-test-insights-privacy-selector'],
```

Comment:
I think, in this case the add-on won't be able to strip the attribute out. Here's how the documentation suggests to do it: https://github.com/simplabs/ember-test-selectors#usage-in-computed-properties

Reply:
Neat! This is more straightforward too. Thanks! Change made